### PR TITLE
imprint-34 Github organisation page.

### PR DESCRIPTION
**Pull request by Max-Levitskiy**
Loading spanner

### CODE REVIEW

File: web/src/app/github/github-organisation-page/github-organisation-page.component.html [hunk 1]
```diff
@@ -1,6 +1,11 @@
-<mat-list *ngIf="orgStats">
-  <mat-list-item *ngFor="let stat of orgStats">
-    <img class="avatar" src="https://github.com/{{stat.name}}.png?size=56" width="56" height="56" alt=""/>
-    <span matBadge="{{stat.imprint.value}}" matBadgeOverlap="false">{{stat.name}}</span>
-  </mat-list-item>
-</mat-list>
+<mat-spinner *ngIf="loading"></mat-spinner>
```
Comment (Tsvetkovv) [inline web/src/app/github/github-organisation-page/github-organisation-page.component.html:1]:
```suggestion
<mat-spinner *ngIf="loading; else list"></mat-spinner>
```

File: web/src/app/github/github-organisation-page/github-organisation-page.component.html [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+
+<div *ngIf="!loading">
```
Comment (Tsvetkovv) [inline web/src/app/github/github-organisation-page/github-organisation-page.component.html:3]:
```suggestion
<ng-template #list>
```

File: web/src/app/github/github-organisation-page/github-organisation-page.component.html [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+  <mat-list *ngIf="orgStats">
+    <mat-list-item *ngFor="let stat of orgStats">
+      <img class="avatar" src="https://github.com/{{stat.name}}.png?size=56" width="56" height="56" alt=""/>
+      <span matBadge="{{stat.imprint.value}}" matBadgeOverlap="false">{{stat.name}}</span>
+    </mat-list-item>
+  </mat-list>
+  <div *ngIf="!orgStats">No statistics found for the organisation</div>
+</div>
```
Comment (Tsvetkovv) [inline web/src/app/github/github-organisation-page/github-organisation-page.component.html:11]:
```suggestion
</ng-template>
```

File: web/src/app/github/github-organisation-page/github-organisation-page.component.ts [hunk 4]
```diff
@@ -10,6 +10,7 @@ import {ActivatedRoute, Router} from '@angular/router';
 })
 export class GithubOrganisationPageComponent implements OnInit {
   orgStats: GithubOrganisationStats[];
+  loading = true;
```
Comment (Tsvetkovv) [inline web/src/app/github/github-organisation-page/github-organisation-page.component.ts:13]: it's better to set `true` right before the start fetching

Comment (Max-Levitskiy) [reply web/src/app/github/github-organisation-page/github-organisation-page.component.ts:13]: make a sense :)

**Pull request merged**